addons/udes_stock/models/stock_move_line.py

```diff
@@ -411,10 +412,12 @@ def _assert_result_package(self, result_package):
 
     def generate_lot_name(self, lot_names, product):
         """ If required, create a lot and return it's name in a list """
-        confirm_tracking = self.mapped('picking_id.picking_type_id').u_confirm_tracking
+        confirm_tracking = self.mapped('picking_id.picking_type_id').u_scan_tracking
```

Comment:
this assumes that there is only one picking type (either make it explicit or move `u_scan_tracking` into the mapped